src/state/debug-middleware/timing-average-middleware.js

```diff
@@ -0,0 +1,46 @@
+// @flow
```

Comment:
another debug middleware - this one averages the time taken for actions and prints a result